nex/register_common_secure_server_protocols.go

```diff
@@ -106,6 +107,16 @@ func gameSpecificMatchmakeSessionSearchCriteriaChecksHandler(searchCriteria *mat
 	return true
 }
 
+func onAfterAutoMatchmakeWithParamPostpone(_ nex.PacketInterface, _ *matchmakingtypes.AutoMatchmakeParam) {
+	// * This is ugly but I can't work out a better way to do this
+	// * Set Splatfest rooms to open participation
+	for _, session := range common_globals.Sessions {
+		if session.GameMatchmakeSession != nil && session.GameMatchmakeSession.GameMode.Value == 12 {
+			session.GameMatchmakeSession.OpenParticipation = types.NewPrimitiveBool(true)
```

Comment:
iirc you can just set `session.GameMatchmakeSession.OpenParticipation.Value` to true, might be a smidge faster

Reply:
Unfortunately not:
```
# github.com/PretendoNetwork/splatoon/nex
nex/register_common_secure_server_protocols.go:115:53: cannot use true (untyped bool constant) as *"github.com/PretendoNetwork/nex-go/v2/types".PrimitiveBool value in assignment
```

You might be thinking of the type refactor; https://github.com/PretendoNetwork/nex-go/pull/56 , which we aren't targeting for this server yet.

Reply:
Oh, `.Value`, I can read. That compiles okay, actually.